### DISCUSSION TITLE
new OTP component for code verification (email verification, password reset, MFA etc...)

### DISCUSSION THREAD

**Discussion by Mdev303** [saadeghi/daisyui#2870] [Ideas / feature requests]
Is it possible to have a OTP component like vuetify for handling verification codes ? 
https://vuetifyjs.com/en/components/otp-input/

**Comment by saadeghi** [comment]
It would need JS to navigate through each input after typing or deleting.  
You can use a NPM package for that and apply daisyUI styles to the input so it matches your website.

Let me know if you have any questions
